data-prepper-plugins/kafka-plugins/src/main/java/org/opensearch/dataprepper/plugins/kafka/configuration/TopicConfig.java

```diff
@@ -38,6 +39,11 @@ public class TopicConfig {
     static final Integer DEFAULT_NUM_OF_WORKERS = 2;
     static final Duration DEFAULT_HEART_BEAT_INTERVAL_DURATION = Duration.ofSeconds(5);
 
+
+    private static final Integer NUM_OF_PARTITIONS = 3;
+    private static final Short REPLICATION_FACTOR = 1;
+
```

Comment:
Nit: rename NUM_OF_PARTITIONS to DEFAULT_NUM_OF_PARTITIONS
       rename REPLICATION_FACTOR to DEFAULT_REPLICATION_FACTOR

Reply:
changes have been incorporated.